guides/delivery/setting-up-a-new-project.md

```diff
@@ -0,0 +1,88 @@
+
+There are several things that delivery managers should do first, regardless of the shape of the project:
+
+### Do some reading 
+
+Get hold of any background materials, such as proposal documents, meeting notes, project documents or email chains. Familiarise yourself with the brief and conversations that have taken place so far
+
+### Speak to the person who has had the most involvement in the work to date 
+
+Find out who led the sales bid, or who has the most knowledge about the project at dxw. This should give you a fairly good understanding of what the project is, what work has been done so far and what dxw is expected to deliver. Find out who’s who in the client organisation, and ask for an introduction to the main point of contact there.
+
+### Meet the client team 
+
+At the earliest opportunity, arrange a meeting with the client. It might be one person at this stage or everyone that will be involved in the project. Use the time to build rapport and get to know each other, as well as agree a start date and schedule any important meetings that need to be held in the first few weeks. If you can, bring along other team members as it might be useful to have their input.
+
+### Ask lots of questions
+
+Whether you’re talking to the person who led the sales bid, or the client team, ask questions. 
+
+- What is the brief? What outputs or outcomes are we expected to deliver?
+- Who is the service manager or product owner in the client team? 
+- Who are the important stakeholders we need to be aware of?
+- Who from the client team will be working full-time on the project? 
+- Are there any deadlines, dependencies or risks that the team need to be aware of?
+- What skills and capabilities do the client team have? Are they used to working in agile teams? Have they delivered a digital service before? Are they familiar with user research?
+
+## Setting up the project
+
+Before the project starts there’s also a number of tasks, mostly administrative ones, that need to be done. Below is a checklist of things that need to be in place for a typical project.
+
+- Send out calendar invitations for regular ceremonies:
+  - Standups - daily, 10-15 minutes between 10:00-11:00 (ideally after the dxw company standup)
+  - Sprint planning meetings - fortnightly, for anything between 1-3 hours, usually on the first day of a sprint (a Wednesday)
+  - Retrospectives - fortnightly, usually held at the end of a sprint or before planning
+  - Show and tells - fortnightly, at the end of a sprint for most projects. The client can help you identify a suitable time and place (we recommend keeping it to a regular time to maximise attendance) 
+- Create a Slack channel and add dxw and client team members. Clients and contractors should be set up as single-channel guests
```

Comment:
Given that several of these items have multiple sentences in them, what do you think about adding a period at the end of all of them?

Reply:
I asked about this in the #helpmedesign Slack channel (https://dxw.slack.com/archives/C8Q0WJ10S/p1529918540000506) because I wasn't sure on what our style for this is. It looks like we're conflicted and we don't have an agreed style for punctuating bullet points.

To try to un-block this, I looked through the rest of the Playbook to see what we do for other bullet point lists. I found some bullet point lists that are punctuated and others that aren't (sigh).

I'm going to go with Gaz's advice that if they're meant to be read as individual sentences then they should be capitalised and have full stops (periods). Will make that change now. Thanks for pointing out.